### DISCUSSION TITLE
Get the list of stored recordings with a file size property for every one

### DISCUSSION THREAD

**Discussion by abouddi** [AxisCommunications/discussions#68] [VAPIX] [answered]
I know that there is a way to get the file size of a single recording using this call as an example
axis-cgi/record/export/properties.cgi?schemaversion=1&recordingid=20141211_121153_025B_00408CCAD5D78&diskid=SD_DISK

I wonder if there is any way to get all the recordings stored with a file size property for every recording. I am looking for a similar output to this call but with a file size property
axis-cgi/record/list.cgi?recordingid=all

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @abouddi. Unfortunately the documentation of [list recordings](https://www.axis.com/vapix-library/subjects/t10175981/section/t10004596/display?section=t10004596-t10004652) doesn't seem to indicate that this feature exists.
